sdk/attestation/azure-security-attestation/README.md

```diff
@@ -1,39 +1,362 @@
+
+```bash
+cmake .. -DCMAKE_BUILD_TYPE=Debug
+cmake --build .
+```
+Then you can consume the built library with the header files.
+make/ninja install is work in progress.
+
+
+### Via NuGet
+WIP
+TODO when ready.
+
+### Via vcpkg
+The easiest way to acquire the C++ SDK is leveraging vcpkg package manager. See the corresponding [Azure SDK for C++ readme section][azsdk_vcpkg_install].
+
+To install Azure Storage packages via vcpkg:
+
+```cmd
+> vcpkg install azure-security-attestation-cpp
+```
+
+Then, use in your CMake file:
+
+```CMake
+find_package(azure-security-attestation-cpp CONFIG REQUIRED)
+target_link_libraries(<your project name> PRIVATE Azure::azure-security-attestation)
+
+```
+
+## Dependencies
+
+  - [Azure Core SDK](https://github.com/Azure/azure-sdk-for-cpp/blob/main/README.md)
+  - OpenSSL
+
+
+
+### Authenticate the client
+
+Many of the APIs supported by the Azure Attestation service require authentication (some do not, if an API does not require
+authentication, the documentation for that API will reflect that the attestation service instance does not require authentication).
+
+To interact with the authenticated APIs supported by the Azure Attestation service, your client must present an Azure Active Directory bearer token to the service.
+
+The simplest way of providing a bearer token is to use the  `DefaultAzureCredential` authentication method by providing client secret credentials is being used in this getting started section, but you can find more ways to authenticate with [azure-identity][azure_identity].
 
 # Key concepts
 
-Bullet point list of your library's main concepts.
+The Microsoft Azure Attestation service runs in two separate modes: "Isolated" and "AAD". When the service is running in "Isolated" mode, the customer needs to
+provide additional information beyond their authentication credentials to verify that they are authorized to modify the state of an attestation instance.
+
+There are four major client types provided in this SDK:
+
+- [SGX and TPM enclave attestation.](#attestation)
+- [MAA Attestation Token signing certificate discovery and validation.](#attestation-token-signing-certificate-discovery-and-validation)  
+- [Attestation Policy management.](#policy-management)
+- [Attestation policy management certificate management](#policy-management-certificate-management) (yes, policy management management).
+
+Each attestation instance operates in one of three separate modes of operation:
+
+- Isolated mode.
+- AAD mode.
+- Shared mode.
+
+## Isolated Mode Attestation Instances.
+In "Isolated" mode, the customer indicates that they want to ensure that Microsoft administrators cannot influence the inputs or outputs of an attestation service instance. When the attestation service instance is running in Isolated mode, the customer is expected to provide additional proof that they are authorized to make changes to the attestation service instance.
```

Comment:
nit: be consistent, add new line after heading

There's a markdown linter extension in VS Code which helps point out such formatting or other linting issues, if you want to use that while updating the doc :)